nixos/modules/services/web-apps/healthchecks.nix

```diff
@@ -143,8 +151,34 @@ in
             '';
             apply = boolToPython;
           };
+
+          DB = mkOption {
+            type = types.enum [ "sqlite" "postgres" "mysql" ];
+            default = "sqlite";
+            description = lib.mdDoc "Database engine to use.";
+          };
+
+          DB_NAME = mkOption {
+            type = types.str;
+            default =
+              if settings.config.DB == "sqlite"
+              then "${cfg.dataDir}/healthchecks.sqlite"
+              else "hc";
```

Comment:
Was that the default before? I think using healthcheck would be more descriptive.

Reply:
old default is explained in the commit message https://github.com/NixOS/nixpkgs/pull/251323/commits/ef9725f01fcc7cd49bb14e13e5ee43e1f687ab5b

Reply:
No, there was no default because sqlite path was used as the default value even if DB was postgres / mysql. I took this default from Healthchecks documentation (see https://healthchecks.io/docs/self_hosted_configuration/#DB_NAME).

What is the convention in nixpkgs: to use a better default value or to use the package's defaults?

Reply:
> What is the convention in nixpkgs: to use a better default value or to use the package's defaults?

usually we stick to upstream unless there are good reasons not to. Both are probably fine here.